### PR TITLE
Attachment of MNE Demo notebook

### PR DESCRIPTION
**Pull request by LoganJF**

### CODE REVIEW

File: ptsa/data/TimeSeriesX.py [hunk 1]
```diff
@@ -106,9 +123,6 @@ def to_hdf(self, filename, mode='w'):
             raise RuntimeError("You must install h5py to save as HDF5")
 
         with h5py.File(filename, mode) as hfile:
-            hfile.attrs['ptsa_version'] = ptsa_version
```
Comment (mivade) [inline ptsa/data/TimeSeriesX.py]: It looks like you removed the `ptsa_version` and `created` attributes which is causing the TravisCI build to fail. Can you add these back?

**Comment by mivade** [comment]
@ctw do you have any comments on the function to load EDF files?

**Comment by mivade** [comment]
@LoganJF looks good so far. I commented above about fixing the existing test failure. It would also be nice to add tests for the new functionality (I can help you with that).

**Comment by LoganJF** [comment]
I did a bad job describing the updates since I accidentally committed to my own branch at first. In the demo section there should be a ipynb which contains information about reading in either raw biosemi eeg path (e.g. .bdf file) into mne, and a more general way to load them after loading the events. Additionally, in TimeSeriesX there are two new methods, one for reading EDFs, and one for a fix of xr.concat. 

Christoph didn't like that the load EDF function 1) Is attached as a method to TimeSeries rather than a separate script (e.g. He thinks organization wise it would be much better to have a script EDFReader.py with the function) and 2) ptsa shouldn't unnecessarily use external packages. I think 2 is less of an issue if we're going to include more MNE functionality as a whole, but I completely agree with point 1 and could rewrite EDF loading as a script if desired. I'm happy to 

Additionally, the concat function should be redone such that rather than checking for an 'events' dimension, it checks the type of all dimensions and for those that are numpy record arrays resets them afterwards.

**Pull request closed without merging**